enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1334 @@
+
+If no `startTime` or `duration` is specified, any day selected by the RRULE will suggest a
+permissive 24h window unless a date is in the `exclude` ranges.
+
+**RRULE Constraints**
+A valid RRULE for change management:
+- must identify a date, so, although RRULE supports `FREQ=HOURLY`, it will be rejected if an attempt it made to use it.
+- cannot specify an end for the pattern. `RRULE:FREQ=DAILY;COUNT=3` suggests
+  an event that occurs every day for three days only. As such, neither `COUNT` nor `UNTIL` is 
+  supported and will be rejected if an attempt is made to use them.
+- cannot specify a permissive window more than 2 years away.
+
+**Overview of Interactions**
+The MaintenanceSchedule strategy, along with `<resource>.spec.changeManagement.pausedUntil` allows a cluster lifecycle administrator to express 
+one of the following:
+
+| pausedUntil    | permit | exclude | Enforcement State (Note that **effective** state must also take into account hierarchy)                                                                                                                                                                                  |
+|----------------|--------|---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `nil` | `null` | `null`  | Restrictive indefinitely           |
+| `nil` | set    | `null`  | Permissive during reoccurring windows time. Paused at all other times.                                                                                                                  |
+| `nil` | set    | set     | Permissive during reoccurring windows time modulo excluded date ranges during which it is paused. Paused at all other times.                                                                                                                                                                                                         |
+| `nil` | `null` | set     | Permissive except during excluded dates during which it is paused.                                                                                                                             |
+| date           | *      | *       | Honor permit and exclude values, but only after the specified date. For example, permit: `null` and exclude: `null` implies the strategy is indefinitely permissive after the specified date. |
+| `true`         | *      | *       | Paused/restrictive indefinitely |
+
+
+#### CLI Assisted Rollout Scenarios
+Once this proposal has been implemented, it is expected that `oc` will be enhanced to permit users to 
+trigger assisted rollouts scenarios for worker-nodes (i.e. where they control the timing of a rollout
+completely but progress is made using MCO automation). 
+
+`oc adm update` will be enhanced with `worker-node` subverbs to initiate and pause MCO work. 

```

Comment:
This would cover infra nodes too? What about other classes of nodes that we define arbitrarily? How does the user break down to various MCP levels?

Reply:
It would cover infra. The proposed verbs act on MCPs, so a user would be able to act separately on `infra` vs other worker MCPs. If a user did want further granularity, they would move into the `Manual` realm where they are free to script drains/reboots based on any criteria.